.github/workflows/size-compare.yml

```diff
@@ -0,0 +1,68 @@
+name: Compare Sizes
+
+on:
+  pull_request:
+
+jobs:
+  compare:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Wait for ${{github.base_ref}} build to succeed
+        uses: fountainhead/action-wait-for-check@v1.1.0
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏very cool! But could we instead do this in the `build` step?

Right now this CI job would kick off in parallel with all the others. And then it would just sit and wait. This means we're wasting CI resources with just waiting.

It's not really a problem _right now_ as we're on the FREE github plan, but I would also like to respect them and not abuse the free plan.

Reply:
Good idea!